crates/polars-stream/src/async_executor/mod.rs

```diff
@@ -313,6 +316,37 @@ impl<'scope, 'env> TaskScope<'scope, 'env> {
     }
 }
 
+pub struct ExecutorScope<'scope, 'env>(TaskScope<'scope, 'env>);
```

Comment:
I don't think this is sound, please remove the changes to `async_executor`. Try to use the `scope` argument given to you by `spawn` to spawn tasks associated with an execution phase. If you need tasks to run on the CPU-intensive pool but they last longer than a phase, please rebase on https://github.com/pola-rs/polars/pull/18163.

Reply:
Have updated to use `async_executor::spawn`